docs/docs/reference/crd-reference/taskdefinition.md

```diff
@@ -396,179 +398,180 @@ file.
 
 ## Examples for deno-runtime runner
 
-### Example 1: inline script for a Deno script
-
-This example defines a full-fledged Deno script
-within the `KeptnTaskDefinition` YAML file:
-
-```yaml
-apiVersion: lifecycle.keptn.sh/v1beta1
-kind: KeptnTaskDefinition
-metadata:
-  name: hello-keptn-inline
-spec:
-  deno:
-    inline:
+=== "Example 1: inline script for a Deno script"
+
+    This example defines a full-fledged Deno script
+    within the `KeptnTaskDefinition` YAML file:
+
+    ```yaml
```

Comment:
Let's please move all the yaml or code snippets to a separate file and link them via include. Same as done [here](https://github.com/keptn/lifecycle-toolkit/pull/3005/files#diff-fc8617bb6853a0241142d24001ab35b560255f3a7adb0962f615f41c5b453d0fR140)

Reply:
Thanks for giving the example @odubajDT .

I created `examples` directory inside docs/docs/assets/crd/, moved all the code snippets there and linked them via include.